docs/sources/write/shortcodes/index.md

```diff
@@ -333,6 +333,121 @@ compactor:
 
 {{< /collapse >}}
 
+## Guide
+
+The `guide` shortcode embeds a Grot guide into the docs that people can follow to get help and guidance on topics and direction to documentation resources.
+
+```markdown
+{{</* guide name="instrument" title="title" text="text" button="button" */>}}
```

Comment:
https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#placeholder-variables

```suggestion
{{</* guide name="<NAME>" title="<TITLE>" text="<TEXT>" button="<BUTTON>" */>}}
```